### PR TITLE
Handle multiple cron syntax

### PR DESCRIPTION
**Pull request by mattdavis0351**
Adding an array of acceptable answers should allow us to quickly check for different cron syntax.  If the current file push works in testing then we can add the other arrays and commit these changes.

This pull request address [this issue](https://github.com/githubtraining/lab-scheduled-events/issues/3)

### CODE REVIEW

**Comment by hectorsector** [comment]
Closes https://github.com/githubtraining/lab-scheduled-events/issues/3

File: main.js [hunk 1]
```diff
@@ -1,46 +1,48 @@
...
+          `${process.env.GITHUB_WORKSPACE}/.github/workflows/${file}`,
+          "utf8"
         )
-      )
+      );
+
       // TODO: if desired keys dont' exist prevent failure but provide feedback
-      if (answers[filename].trim() === doc.on.schedule[0].cron.trim()) {
-        console.log(`Results for ${filename}: correct`)
+      if (answers[filename].trim().includes(doc.on.schedule[0].cron.trim())) {
```
Comment (hectorsector) [inline main.js]: @mattdavis0351 I believe this line is throwing an error ([source](https://github.com/hectorsector/improved-octo-tribble/runs/2004227048?check_suite_focus=true#step:3:7)):
```
Error: TypeError: answers[filename].trim is not a function
```

**Review by hectorsector — CHANGES_REQUESTED** [review]

File: main.js (hunk 1 above)
Comment (mattdavis0351) [reply main.js]: Doh!  I've made an update 🤦 .  This is why we need testing

**Comment by mattdavis0351** [comment]
@hectorsector Go ahead and try now.  I've also added some unit tests to hopefully help with any future changes and to begin establishing some best practices.

**Review by hectorsector — APPROVED** [review]
Works for me, @mattdavis0351!

**Pull request merged**